src/System.Drawing.Common/src/Resources/Strings.resx

```diff
@@ -343,4 +402,10 @@
   <data name="BlendObjectLastElementInvalid" xml:space="preserve">
     <value>Invalid Blend object. The positions array must have 1.0 as its last element.</value>
   </data>
+  <data name="DllNotFoundExceptionMessage" xml:space="preserve">
```

Comment:
Ideally we would add overloads to DNFE and EPNFE that take the dll/entry point name. But this is fine for now.